source/common/tls/utility.cc

```diff
@@ -419,6 +419,22 @@ Envoy::Ssl::ParsedX509NamePtr Utility::parseSubjectFromCertificate(X509& cert) {
   return parseX509NameFromCertificate(cert, CertName::Subject);
 }
 
+absl::optional<uint64_t> Utility::getExpirationUnixTime(const X509* cert) {
+  if (cert == nullptr) {
```

Comment:
You can simplify some code by never passing nullptr to this, removing this check, and make the result not be optional.

Reply:
Was following the pattern in this file of handling nullptr inside the function and return `absl::optional<...>`. 
Perhaps we can keep it like `absl::optional<...>` for consistency with other methods like `getDaysUntilExpiration`, ? let me know if you disagree.